bugbear.py

```diff
@@ -2045,6 +2102,8 @@ def visit_Lambda(self, node):
         " statement."
     )
 )
+
 B950 = Error(message="B950 line too long ({} > {} characters)")
 
+B999 = Error(message="B999 editing loop iterable can cause unintended bugs!")
```

Comment:
```suggestion
B999 = Error(message="B038 editing a loop's mutable iterable often leads to unexpected results/bugs")
```
I'm not tied to this, but just tried to be as explicit and descriptive as possible

Reply:
Sounds good!